style.css

```diff
@@ -0,0 +1,48 @@
+*,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+body {
+  width: 100%;
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
+#title {
+  text-align: center;

```

Comment:
- [ ] Good job so far 👍  however you can use a separate class for `text-align:  center;` to make your code more DRY ❗ 😄